### PR TITLE
📖 Mostly spelling fixes

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
Also some indentation fixes and quoting fixes.

## Summary
This PR fixes some spelling misteaks, adds some missing words to the wordlist, and makes some fixes to indenting and quoting (so that the spell-checker will properly recognize quoted code blocks --- I hope).

Website preview at https://mikespreitzer.github.io/kcp-edge-mc/doc-speling-1011/readme/

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/direct/architecture.md [hunk 1]
```diff
@@ -472,7 +472,7 @@ Relation "USES": has a row whenever the `Binding`'s list of workload objects use
 
 | column name | type | in key |
 | ----------- | ---- | ------ |
-| bindingName | string | yes |
+| `bindingName` | string | yes |
 | gr | metav1.GroupResource | yes |
```
Comment (vMaroon) [inline docs/content/direct/architecture.md:476]: Shouldn't "gr" also get wrapped?

File: docs/content/direct/core-chart.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ This can be:
 
     If a host port number different from the expected 9443 is used for the Kind cluster, then the same port number must be specified during the chart installation by adding the following argument `--set "kubeflex-operator.externalPort=<port>"`.
 
-    By default the KubeStelalr Core chart uses a test domain `localtest.me`, which is ok for testing on a single host machine. However, scenarios that span more than one machine, it is necessary to set `--set "kubeflex-operator.domain=<domain>"` to a more appropriate `<domain>` that can be reached from Workload Execution CLusters (WECs).
+    By default the KubeStellar Core chart uses a test domain `localtest.me`, which is OK for testing on a single host machine. However, scenarios that span more than one machine, it is necessary to set `--set "kubeflex-operator.domain=<domain>"` to a more appropriate `<domain>` that can be reached from Workload Execution Clusters (WECs).
```
Comment (vMaroon) [inline docs/content/direct/core-chart.md:40]: Missing `for` before `scenarios`.

File: docs/content/direct/architecture.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/direct/architecture.md]: Done

**Review by vMaroon — COMMENTED** [review]
/lgtm

**Pull request merged**